### DISCUSSION TITLE
IPad Pro

### DISCUSSION THREAD

**Discussion by nanaasiedu** [excalidraw/excalidraw#3808] [Q&A]
Hello everyone

I need to use Excalidraw for an interview and I would like to use my IPad to draw diagrams.
However I noticed Excalidraw doesn't have palm rejection and always draws dots when I to move the page around
Also the undo button is not visible on the IPad making it extremely difficult to delete little mistakes :/
I'm not sure if this is due to using Excalidraw on a web broswer which doesn't support Apple Pencil?

Am I using it wrong?
How can I use Excalidraw with the IPad Pro to draw diagrams

Thanks!

**Comment by davosian** [comment]
I have run into the same issues on an iPad. For Undo, the only workaround I found is to attach a bluetooth keyboard and use the shortcut cmd-z.

As far as palm rejection goes, I do not know of a way to resolve it other than using a regular computer with a mouse and keyboard (which actually works very well).

**Reply by davosian** [reply]
Btw, the undo problem is already reported as issue: https://github.com/excalidraw/excalidraw/issues/3162

**Reply by dwelle** [reply]
undo/redo will be exposed to the UI in this PR: https://github.com/excalidraw/excalidraw/pull/3832
